### DISCUSSION TITLE
[Feature] view image (from search results) in timeline

### DISCUSSION THREAD

**Discussion by fivestones** [immich-app/immich#4197] [Feature Request]
### The feature

Currently if I search for images and find some results based on a CLIP search, I can easily see each of those pictures. But often I am searching for some specific picture using CLIP not just to see that picture, but all the pictures of the same event or period in time.

What would be great: when I am viewing a photo from a search result (either from a search in the search bar, or even a group of photos shown by clicking a spot on the map), there is some option to "View this photo in the timeline". Or maybe the date field in the info box could be clickable and take me to the timeline for that photo.

Then I'm suddenly in the timeline, and I can see that photo, but surrounding it are all the other photos taken around the same time (instead of just all the other photos that were in the search results.

Thanks!

### Platform

- [ ] Server
- [X] Web
- [X] Mobile

**Comment by bo0tzz** [comment]
Duplicate of #4115

**Reply by fivestones** [reply]
Ahhh, thanks. Didn't see that one.
